### PR TITLE
feat: enable partial graphs in nx console & fix bugs

### PR DESCRIPTION
**Pull request by MaxKless**
- catch errors & use partial graph
- rework graph error display & fixes
- call nx daemon --stop instead of nx reset on refresh
- various fixes
- formatting
- updated error display
- handle broken nx.json
- further intellij fixes
- fix run-gradle.js again
- setColors on wrapped browser load html
- more intellij fixes
- enable partial graph display in nx 19 beta

### CODE REVIEW

File: apps/intellij/src/main/kotlin/dev/nx/console/graph/NxGraphBrowserBase.kt [hunk 1]
```diff
@@ -321,8 +325,9 @@ abstract class NxGraphBrowserBase(protected val project: Project) : Disposable {
               }
             </style>
 
-            <p>Unable to load the project graph. The following error occurred:</p>
-      <pre>${error}</pre>
+            <p>Unable to load the project graph. The following error${if(errors.isNotEmpty()) "s" else ""} occurred:</p>
+      ${errors.map { "<pre>${it.message ?: ""} \n ${it.stack ?: ""}</pre>" }.joinToString("\n")
+        }
       If you are unable to resolve this issue, click here to <a href="#" onclick="window.reset()">reset</a> the graph.
```
Comment (jaysoo) [inline apps/intellij/src/main/kotlin/dev/nx/console/graph/NxGraphBrowserBase.kt:331]: Should we also say to run `nx reset` in the terminal as an alternative? In the case of the `CancellationException` it is the only way to resolve.

File: apps/intellij/src/main/kotlin/dev/nx/console/nxls/NxlsProcess.kt [hunk 2]
```diff
@@ -58,7 +57,7 @@ class NxlsProcess(private val project: Project, private val cs: CoroutineScope)
 
                             logger.trace("Nxls early exit: $this")
 
-                            Notifier.notifyNxlsError(project)
+                            //                            Notifier.notifyNxlsError(project)
```
Comment (jaysoo) [inline apps/intellij/src/main/kotlin/dev/nx/console/nxls/NxlsProcess.kt:60]: We don't want this notifier?

Comment (MaxKless) [reply apps/intellij/src/main/kotlin/dev/nx/console/nxls/NxlsProcess.kt:60]: No, the exit is now triggered in "allowed" circumstances when restarting. We will need to provide a message when the nxls errors out in the future, but this was being triggered wrongly.

Comment (jaysoo) [reply apps/intellij/src/main/kotlin/dev/nx/console/nxls/NxlsProcess.kt:60]: Let's remove it then

**Review by jaysoo — APPROVED** [review]

**Pull request merged**